exporter/exporterhelper/queue_sender.go

```diff
@@ -38,8 +43,15 @@ type QueueSettings struct {
 }
 
 // NewDefaultQueueSettings returns the default settings for QueueSettings.
+//
```

Comment:
Can delete these 2 lines of comment. I prefer only the deprecation message if this is not a long-term deprecated.